Gems/Multiplayer/Code/Source/Debug/MultiplayerConnectionViewportMessageSystemComponent.cpp

```diff
@@ -386,27 +316,25 @@ namespace Multiplayer
             return;
         }
 
-        const AZ::u32 previousState = debugDisplay->GetState();
```

Comment:
Ironically you probably still want to store and restore the previous state :)

We need to update the API to have a push state & pop state functions to make this straightforward, but that can be done later.

Reply:
Good catch. The call to DrawQuad doesn't affect the state, but debugDisplay->DepthWriteOff() and DepthTestOff() does.
 Fixed!